cluster/hooks.go

```diff
@@ -246,19 +240,48 @@ func InstallLogging(input interface{}, param pkgCluster.PostHookParam) error {
 	if err != nil {
 		return err
 	}
-	loggingValues := map[string]interface{}{
-		"bucketName": loggingParam.BucketName,
-		"region":     loggingParam.Region,
-		"secret": map[string]interface{}{
-			"secretName": installedSecretValues.Name,
-		},
-	}
-	marshaledValues, err := yaml.Marshal(loggingValues)
+
+	// Determine the type of output plugin
+	logSecret, err := secret.Store.Get(cluster.GetOrganizationId(), loggingParam.SecretId)
 	if err != nil {
 		return err
 	}
+	switch logSecret.Type {
```

Comment:
Is this secret needed by logging operator at startup? if so shouldn't this be installed into the cluster before the operator deployed?

Reply:
This needed for the output plugin installed separatly